src/js/Controller.js

```diff
@@ -0,0 +1,42 @@
+export default class Controller {
+  constructor(model, views) {
+    this.model = model;
+    this.views = views;
```

Comment:
```suggestion
  model; views;
  constructor(model, views) {
    this.model = model;
    this.views = views;
```
이렇게 미리 멤버변수를 클래스 필드 영역에 정의해주면 어떨까요?
그래야 다른 사람들이 이 클래스에서 사용되는 멤버변수를 파악할때 편할 것 같아요!

```suggestion
  #model; #views;
  constructor(model, views) {
    this.#model = model;
    this.#views = views;
```
그리고 이렇게 `#`을 이용해서 private 으로 멤버변수를 선언할 수 있답니다!